ui/vite.config.ts

```diff
@@ -5,15 +5,16 @@ import { defineConfig } from 'vite'
 // https://vitejs.dev/config/
 export default defineConfig(({ mode }) => {
   const base = mode == 'development' ? '' : '/PREFECT_UI_SERVE_BASE_REPLACE_PLACEHOLDER'
+
   return {
     plugins: [vue()],
     base,
     resolve: {
       alias: [{ find: '@', replacement: resolve(__dirname, './src') }],
       dedupe: ['vue', 'vue-router'],
     },
-    css: {
-      devSourcemap: true,
```

Comment:
removed this because I don't think this is a valid setting anymore plus `sourcemaps: true` should enable all sourcemaps including css.